llvm/lib/Target/EVM/EVMStackify.cpp

```diff
@@ -1013,6 +1037,29 @@ void StackModel::postProcess() {
 
   for (auto *MI : ToErase)
     MI->eraseFromParent();
+
+  for (MachineBasicBlock &MBB : *MF)
+    for (MachineInstr &MI : MBB)
+      stackifyInstruction(&MI);
+
+  auto *MFI = MF->getInfo<EVMMachineFunctionInfo>();
+  MFI->setIsStackified();
+
+  // In a stackified code register liveness has no meaning.
+  MachineRegisterInfo &MRI = MF->getRegInfo();
+  MRI.invalidateLiveness();
+
+  for (MachineBasicBlock &MBB : *MF) {
```

Comment:
Bundle usage is to be explained somewhere in code.

Reply:
Added the description.